.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,3 +16,29 @@ Addresses issue #
 - [ ] My code has proper inline documentation.
 - [ ] I have added a QA Brief on the issue linked above.
 - [ ] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).
+
+## Code Review Checklist
+
+- [ ] **Run the code.** 
+  <details><em>Do not just look at the code (the “minimum” code review), but also briefly test it in your development environment, especially if the pull request touches production code. This should explicitly not be QA, but a basic “sanity check” that the PR is not significantly off or even broken.</em></details>
```

Comment:
Why wrap all the explanation in `em`? Is there a problem with the formatting we'd get otherwise?

Reply:
Just wanted to differentiate checklist titles and descriptions. Anyway removed descriptions.